web-app/README.md

```diff
@@ -1,3 +1,195 @@
-# Cellular Connected Electronic Kiosk Web App
+# Cellular-Connected Electronic Kiosk Web App
 
-tbd
\ No newline at end of file
+The Cellular-Connected Electronic Kiosk web app is intended to be an example for how a web app could be built with a greater focus on the acts of compressing the app and zipping it up, uploading it to a cloud, and then using the Notecard to download the ZIP file via cellular, unpack it, and run the app locally on the Raspberry Pi.
+
+- [Cellular-Connected Electronic Kiosk Web App](#cellular-connected-electronic-kiosk-web-app)
+  - [Brief Demo Web App Overview](#brief-demo-web-app-overview)
```

Comment:
I think this could be shortened to "Web App Overview" or just "Overview".

Reply:
Good suggestion, shortened